types/shopify-buy.d.ts

```diff
@@ -0,0 +1,618 @@
+// Type definitions for shopify-buy 2.10
+// Project: http://shopify.github.io/js-buy-sdk/api/
+// Definitions by: Martin Köhn <https://github.com/openminder>
+//                 Stephen Traiforos <https://github.com/straiforos>
+//                 Rosana Ruiz <https://github.com/totemika>
+//                 Juan Manuel Incaurgarat <https://github.com/kilinkis>
```

Comment:
Instead of maintaining our own copy, we should use the types package offered by the same authors: https://www.npmjs.com/package/@types/shopify-buy

```bash
yarn add -D @types/shopify-buy
```

Reply:
@chrischan325 this doesn't seem to be resolved?

Reply:
I added the package but do you mean to add and delete this file?

Reply:
The issue with this types package is that the cart url field is outdated. In the type definitions it's [checkoutUrl](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ab1cf8125a365a3701c717e7305d51ae240ccde3/types/shopify-buy/index.d.ts#L303) but according to the docs it should be should be [webUrl](https://shopify.dev/api/storefront/reference/checkouts/checkout#:~:text=was%20last%20updated.-,webUrl%20(URL!,-))

Reply:
for some reason on the [github issue](https://github.com/Shopify/js-buy-sdk/issues/770#:~:text=for%20current%20cart%0A%20%20%20%20%20*/-,webUrl%3A%20string%3B,-orderStatusUrl%3F%3A%20string%3B%0A%20%20%20%20taxExempt) i found earlier it's correct but on the actual types package its not